brainframe/cli/commands/uninstall.py

```diff
@@ -0,0 +1,74 @@
+import shutil
+from argparse import ArgumentParser
+
+import i18n
+from brainframe.cli import config, docker_compose, os_utils, print_utils
+
+from .utils import command, subcommand_parse_args
+
+
+@command("uninstall")
+def uninstall():
+    install_path = config.install_path.value
+    data_path = config.data_path.value
+
+    args = _parse_args()
+
+    # This command has to be run as root because some BrainFrame services write files
+    # as the root user.
+    if not os_utils.is_root():
+        print_utils.fail_translate("general.user-not-root")
+
+    docker_compose.assert_installed(install_path)
+
+    if args.noninteractive:
+        delete_data = args.delete_data
+    else:
+        delete_data = print_utils.ask_yes_no("uninstall.ask-delete-data")
```

Comment:
I wonder if it would be best to have a `--force` flag if there's data to delete, even for non-interactive?

Reply:
In the case of non-interactive mode, it defaults to not deleting data. It will only delete data if you provide the flag for doing that or answer "yes" to this prompt. I think that's pretty safe considering it prompts you again in interactive mode to confirm what you're doing.